docker2fl/src/main.rs

```diff
@@ -87,7 +88,13 @@ async fn main() -> Result<()> {
     let meta = fungi::Writer::new(&fl_name).await?;
     let store = parse_router(&opts.store).await?;
 
-    let res = docker2fl::convert(meta, store, &docker_image, credentials).await;
+    let container_name = Uuid::new_v4().to_string();
+    let docker_tmp_dir = tempdir::TempDir::new(&container_name).unwrap();
+    let docker_tmp_dir_path = docker_tmp_dir.path().to_owned();
```

Comment:
Why can't u pass the TempDir around instead of crating an owned PathBuf. Or maybe just pass the Path around instead. 
the problem with taking an owned copy of the path is that TempDir can go out of scoop and the directory is deleted but the PathBuf (since it's owned) will remain to be valid. But it will point to a location that doesn't exist anymore. 

This error can be caught by the compiler if:
- You just path the temp dir around, this directory will then remain alive as long as temp dir is alive, once TempDir goes out of scoop it will get automatically deleted.
- You can pass docker_tmp_dir.path() (just the Path) which is a pointer, and hence its life time is linked to the TempDir. Which means compiler will complain if u continue to use the Path if TempDir went out of scoop